### ISSUE TITLE
[BE] Fix method Facebook working after deploy

### ISSUE THREAD

**Issue by minatisleeping** [PiedTeam/NikeCloneTraining-BE-Project#140]
fix cho fb chay dc, chị Minh nói đơn giản lắm :( but i dont think so..

**Comment by minatisleeping** [comment]
Hi there,

The following may not be a bug, I have done my best to follow the available documentation, but if there is a correct way to do this then please let me know.
I am trying to use [naikila](https://naikila-fe.web.app) with Nike deployed by firebase and it not really work, in local still work fine 

## **Expected**: 
If login with FB success, fe will receive url direct to domain/user/login-success/?access_token=...&new_user=...&iat=...&exp=... 
like this: "http://localhost:3000/user/login-success/?access_token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiNjY1ZWM3NTUzYjEwOTg0ODE4MzAzM2FkIiwidG9rZW5fdHlwZSI6MCwic3RhdHVzIjowLCJpYXQiOjE3MTc3MzIyMzksImV4cCI6MTcxODE2NDIzOX0.vOirtdx6_x72qrmQP0Ka-XRHzibGvUFKpuEY885jUYo&new_user=false&iat=Tue%20Jun%2004%202024%2014:50:45%20GMT+0700%20(Indochina%20Time)&exp=Thu%20Jul%2004%202024%2014:50:45%20GMT+0700%20(Indochina%20Time)#_=_"

## How to reproduce

### **Here is my vid I tested on my local**

https://github.com/PiedTeam/NikeCloneTraining-BE-Project/assets/121481596/28b3b7da-8329-42ad-819e-c88d8bfb23af

### **Here is my vid I tested on [naikila](https://naikila-fe.web.app) deployed** 

https://github.com/PiedTeam/NikeCloneTraining-BE-Project/assets/121481596/a3c5b32c-dce3-48c8-b704-7d2850f947eb

In [app dev](https://developers.facebook.com/) I have already configured like this:

![Image](https://github.com/PiedTeam/NikeCloneTraining-BE-Project/assets/121481596/d3570b2d-96a8-4f84-ba46-774f05b4a3b9)

![Image](https://github.com/PiedTeam/NikeCloneTraining-BE-Project/assets/121481596/d22069c8-b4d5-40f0-8959-2e3bd5645971)

**In Summary:** 
- I think I have configured wrong in somewhere..
- The redirect URL to add in your Facebook application configurations" doesn't actually represent the redirect address that is used 

Any help or suggestions for how to resolve this would be very much appreciated.

**Comment by tientran1234** [comment]
I think your url return in env file incorrect you should try check it
